frontend/pages/SoftwarePage/SoftwareVersions/SoftwareVersionsTableConfig.tsx

```diff
@@ -0,0 +1,292 @@
+              ? vulnerabilities[0].cve
+              : `${vulnerabilities.length} vulnerabilities`}
+          </span>
+          <ReactTooltip
+            effect="solid"
+            backgroundColor="#3e4771"
+            id={`vulnerabilities__${cellProps.row.original.id}`}
+            data-html
+          >
+            <span className={`vulnerabilities tooltip__tooltip-text`}>
+              {tooltipText}
+            </span>
+          </ReactTooltip>
+        </>
+      );
+    },
+  };
+};
+
+const generateTableHeaders = (
+  router: InjectedRouter,
+  isPremiumTier?: boolean,
+  isSandboxMode?: boolean,
+  teamId?: number
+): Column[] => {
+  const softwareTableHeaders = [
+    {
+      title: "Name",
+      Header: (cellProps: IHeaderProps): JSX.Element => (
+        <HeaderCell
+          value={cellProps.column.title}
+          isSortedDesc={cellProps.column.isSortedDesc}
+        />
+      ),
+      disableSortBy: false,
+      accessor: "name",
+      Cell: (cellProps: IStringCellProps): JSX.Element => {
+        const { id, name } = cellProps.row.original;
+
+        const onClickSoftware = (e: React.MouseEvent) => {
+          // Allows for button to be clickable in a clickable row
+          e.stopPropagation();
+          router?.push(PATHS.SOFTWARE_VERSION_DETAILS(id.toString()));
+        };
+
+        return (
+          <LinkCell
+            path={PATHS.SOFTWARE_VERSION_DETAILS(id.toString())}
+            customOnClick={onClickSoftware}
+            value={name}
+          />
+        );
+      },
+      sortType: "caseInsensitive",
+    },
+    {
+      title: "Version",
+      Header: "Version",
+      disableSortBy: true,
+      accessor: "version",
+      Cell: (cellProps: IVersionCellProps): JSX.Element => (
+        <TextCell value={cellProps.cell.value} />
+      ),
+    },
+    {
+      title: "Type",
+      Header: "Type",
+      disableSortBy: true,
+      accessor: "source",
+      Cell: (cellProps: IStringCellProps): JSX.Element => (
+        <TextCell formatter={formatSoftwareType} value={cellProps.cell.value} />
+      ),
+    },
+    {
+      title: "Vulnerabilities",
+      Header: (cellProps: IHeaderProps): JSX.Element => (
+        <HeaderCell
+          value={cellProps.column.title}
+          disableSortBy={false}
+          isSortedDesc={cellProps.column.isSortedDesc}
+        />
+      ),
+      disableSortBy: false,
+      // the "vulnerabilities" accessor is used but the data is actually coming
+      // from the version attribute. We do this as we already have a "versions"
+      // attribute used for the "Version" column and we cannot reuse. This is a
+      // limitation of react-table.
+      // With the versions data, we can sum up the vulnerabilities to get the
+      // total number of vulnerabilities for the software title
+      accessor: "vulnerabilities",
+      Cell: (cellProps: IVulnCellProps): JSX.Element => (
+        <VulnerabilitiesCell vulnerabilities={cellProps.cell.value} />
```

Comment:
> the "vulnerabilities" accessor is used but the data is actually coming
> from the version attribute. 

Doesn't `cellProps.cell.value` point to the data key that corresponds to the accessor? Meaning that the the value will be comming from the `data.vulnerabilities` rather than `data.versions`?

Reply:
yes. there comment was meant for the software titles table config vulnerabilities column. accidentally placed it here